### PR TITLE
Standardize hipBLASLt documentation builds

### PR DESCRIPTION
**Pull request by samjwu**
Summary of proposed changes:
- Updates the readthedocs documentation to use the rocm-docs-core template

Rationale:
 - https://github.com/RadeonOpenCompute/rocm-docs-core#purpose
 - https://confluence.amd.com/display/MLSE/Docs+as+Code+Transition
 - https://confluence.amd.com/display/MLSE/Documentation+RACI+Page

Successful build: https://readthedocs.com/projects/advanced-micro-devices-hipblaslt/builds/1359965/

Preview: https://advanced-micro-devices-hipblaslt.readthedocs-hosted.com/en/rtd/

### CODE REVIEW

**Review by saadrahim — APPROVED** [review]

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+from rocm_docs import ROCmDocs
+
+docs_core = ROCmDocs("hipBLAS Documentation")
```
Comment (jichangjichang) [inline docs/conf.py:9]: hipBLASLt Documentation

Comment (samjwu) [reply docs/conf.py]: Thanks for catching that. Fixed in [56f2843](https://github.com/ROCmSoftwarePlatform/hipBLASLt/pull/64/commits/56f2843dbfea966ea2d66c3e7d9cc6482e8db389)

**Comment by saadrahim** [comment]
@jichangjichang can you take a look at this PR and see if you can approve this

**Comment by jichangjichang** [comment]
@samjwu @saadrahim 
Where we can check the built result?
The CI's status is checkmark but the page shows 404.

> docs/readthedocs.com:advanced-micro-devices-hipblaslt — Read the Docs build succeeded!

https://readthedocs.com/projects/advanced-micro-devices-hipblaslt/builds/1365424/

**Review by jichangjichang — APPROVED** [review]

**Comment by samjwu** [comment]
> @samjwu @saadrahim Where we can check the built result? The CI's status is checkmark but the page shows 404.
> 
> > docs/readthedocs.com:advanced-micro-devices-hipblaslt — Read the Docs build succeeded!
> 
> https://readthedocs.com/projects/advanced-micro-devices-hipblaslt/builds/1365424/

You'll want to click the second "View docs" button. The build page is misleading because the first big "View docs" button on the top of the page takes you to the build on the develop branch.

This is the correct link: https://advanced-micro-devices-hipblaslt--64.com.readthedocs.build/en/64/

**Pull request merged**